### DISCUSSION TITLE
Any talks on adding an optional module for a global store like Spruce?

### DISCUSSION THREAD

**Discussion by james0r** [alpinejs/alpine#1231] [3. Feature ideas]
Just tried to demo it but noticed the site and docs are broken. 

Is there any plan for this? I totally like keeping Alpine light but maybe as an additional package could be pretty sick.

Thanks!

**Comment by ryangjchandler** [comment]
The Spruce docs had some DNS issues. They should all be fine now though.

**Reply by ryangjchandler** [reply]
If you need the URL, here it is: https://spruce.ryangjchandler.co.uk.

**Reply by james0r** [reply]
Awesome. Appreciate it. I will play with it tonight.
